app/lib/route-helpers.js

```diff
@@ -11,6 +11,19 @@ const throwIfPreConditionError = (request) => {
   }
 }
 
+const checkUserAccess = async (request, user) => {
```

Comment:
probably leave it for this PR, but can we change the name of this variable to something like...

`getCorrectAccessRoute` - or something that communicates that the function is to get the route.